app/models/recently_added.rb

```diff
@@ -1,39 +1,17 @@
 # frozen_string_literal: true
 
 class RecentlyAdded
-  require 'net/http'
-  require 'json'
-
-  def self.feed(root_path)
-    url = URI.join(root_path, "catalog.json").to_s
-    resp = http_get(url)
-    result = JSON.parse(resp.body)
-    payload = {}
-    result['data'].each do |entry|
-      payload[entry['id']] = {
-        title: entry['attributes']['title_tsim']['attributes']['value'],
-        link: entry['links']['self'],
-        author: entry['attributes']['author_tesim']['attributes']['value'],
-        genre: entry['attributes']['genre_ssim']['attributes']['value'],
-        issue_date: entry['attributes']['issue_date_ssim']['attributes']['value']
-      }
-    end
-    payload
+  def self.feed
```

Comment:
Notice that I am querying Solr directly (rather than the `/catalog` endpoint). This allows me to control the order of the data and also allows me to use normal `SolrDocument` objects (with all our methods) rather than a hash where we have to re-implement some of the logic in `SolrDocument`.